spec.md

```diff
@@ -3341,6 +3341,25 @@ command =
         string_dq ,
         { "," , expr }
     , ")" ,
+  | "assert" , "(" ,
+        expr , "," ,
+        expr , "," ,
+        expr , "," ,
+        string_dq ,
+        { "," , expr }
+    , ")" ,
+  | "assume" , "(" ,
+        expr , "," ,
+        expr , "," ,
+        expr , "," ,
+        string_dq
+    , ")" ,
+  | "cover" , "(" ,
+        expr , "," ,
+        expr , "," ,
+        expr , "," ,
+        string_dq
+    , ")" ,
```

Comment:
What does a format String on a cover mean? What is a typical or lowering to SystemVerilog?

Reply:
`string_dq` doesn't indicate a format string. A format string is not currently specially handled in the grammar. `{ "," , expr }` represents operands for substitutions.